src/mapdata/ExitDirection.h

```diff
@@ -32,7 +33,12 @@ NODISCARD extern bool isNESW(ExitDirEnum dir);
 NODISCARD extern bool isUpDown(ExitDirEnum dir);
 NODISCARD extern bool isNESWUD(ExitDirEnum dir);
 NODISCARD extern ExitDirEnum opposite(ExitDirEnum in);
-NODISCARD extern const char *lowercaseDirection(ExitDirEnum dir);
+NODISCARD extern std::string_view lowercaseDirection(ExitDirEnum dir);
+NODISCARD inline const char *lowercaseDirectionC(ExitDirEnum dir)
```

Comment:
Why are we changing the lowest API to support a std::string_view and then wrapping it back to a const char *?

Shouldn't we just keep it as a const char * and then use a std::string_view in the one class where it was needed?

Reply:
I rewrote it like that because the conversion `std::string_view` -> `const char *` is O(1): you just extracts the pointer.
Instead the conversion `const char *` -> `std::string_view` is O(n): it must call `strlen()` or equivalent.

So `std::string_view` is the better alternative to return from such a function: the alternate function returning `const char *` can be defined on top of it.

Also, I could not find a pair of names better than `lowercaseDirection()` and `lowercaseDirectionC()`:
the alternative `lowercaseDirectionStringView` or some abbreviation of it really looked ugly to me.